util/topsql/collector/cpu.go

```diff
@@ -37,7 +37,7 @@ const (
 // Collector uses to collect SQL execution cpu time.
 type Collector interface {
 	// Collect uses to collect the SQL execution cpu time.
-	Collect(stats []SQLCPUTimeRecord)
+	Collect(timestamp uint64, stat []SQLCPUTimeRecord)
```

Comment:
Can we not modify this interface? Because by design we don’t want the caller to have a way to set the timestamp, it can only be set by the reporter according to the time when the data is obtained. We can avoid the data race by modifying the test instead of modifying the main code~

Reply:
I am trying to optimize the test

Reply:
If you cannot set the timestamp, you cannot test the timestamp on the Thread safety.